pages/spotlight/index.js

```diff
@@ -1,26 +1,30 @@
-import { useRouter } from "next/router";
 import Header from "@/components/Header/Header";
-import Image from "next/image";
 import styled from "styled-components";
 import Navigation from "@/components/Navigation/Navigation";
+import BookDetailsCard from "@/components/BookDetailsCard/BookDetailsCard";
 
-const StyledImage = styled(Image)`
-  border-radius: 8px;
-  position: relative;
-  top: -20px;
-  box-shadow: 0 6px 20px 5px rgba(0, 0, 0, 0.19);
-`;
-const StyledBookDetail = styled.article`
-  border: 2px solid black;
-  border-radius: 8px;
-  margin: 2rem auto;
-  text-align: center;
-`;
-const StyledSection = styled.section`
-  display: flex;
-  justify-content: space-around;
-  font-style: italic;
-`;
+export default function Spotlight({ books }) {
+  const currentBook = books[Math.floor(Math.random() * books.length)];
+  return (
+    <>
+      <StyledDetailedPage>
+        <Header />
+        <StyledBookStar>Book ✨ Star</StyledBookStar>
+        <BookDetailsCard
+          cover={currentBook.cover}
+          title={currentBook.title}
+          id={currentBook.id}
+          author={currentBook.author}
+          genre={currentBook.genre}
+          publishYear={currentBook.publishYear}
+          pages={currentBook.pages}
+          description={currentBook.description}
```

Comment:
consider to pass the currentBook object in combination with the object destructuring syntax.